docs/content/integrations/dbt.mdx

```diff
@@ -18,7 +18,7 @@ Dagster orchestrates dbt alongside other technologies, so you can combine dbt wi
 Dagster has built-in support for loading dbt models, seeds, and snapshots as software-defined assets, enabling you to:
 
 - Visualize and orchestrate a graph of dbt assets, and execute them with a single dbt invocation
-- Version your dbt models by their defining SQL code, allowing downstream assets to be automatically marked stale when a model changes
+- Version your dbt models by their defining SQL code, allowing Dagster to indicate when a model has changed
```

Comment:
Just verified that the code version tag still works for dbt models, and it does, super slick!